engine/table/src/main/java/io/deephaven/engine/table/impl/by/PartitionByChunkedOperator.java

```diff
@@ -471,23 +547,25 @@ public void propagateInitialState(@NotNull final QueryTable resultTable) {
             final boolean setCallSite = QueryPerformanceRecorder.setCallsite(callSite);
             try (final ResettableWritableObjectChunk<QueryTable, Values> tablesResettableChunk =
                     ResettableWritableObjectChunk.makeResettableChunk();
-                    final ResettableWritableObjectChunk<WritableRowSet, Values> addedRowSetsResettableChunk =
-                            ResettableWritableObjectChunk.makeResettableChunk();
+                    final ResettableWritableObjectChunk<RowSetBuilderSequential, Values> addedBuildersResettableChunk =
```

Comment:
This method is called exactly once, and it switches us over to `initialized`. I would add an `Assert.neqTrue(initialized, "initialized")` at the top of the method. You can and should remove code here that allows for `initialized == true`.